lib/purescript-ast/src/Language/PureScript/AST/Binders.hs

```diff
@@ -71,100 +71,62 @@ data Binder
 -- the `Ord` instance was needed for the speed-up, but I did not want the `Eq`
 -- to have mismatched behavior.
 instance Eq Binder where
-  (==) NullBinder NullBinder = True
-  (==) NullBinder _ = False
-
-  (==) (LiteralBinder _ lb) (LiteralBinder _ lb') = (==) lb lb'
-  (==) LiteralBinder{} _ = False
-
-  (==) (VarBinder _ ident) (VarBinder _ ident') = (==) ident ident'
-  (==) VarBinder{} _ = False
-
-  (==) (ConstructorBinder _ qpc bs) (ConstructorBinder _ qpc' bs') =
-    (==) qpc qpc' && (==) bs bs'
-  (==) ConstructorBinder{} _ = False
-
-  (==) (OpBinder _ qov) (OpBinder _ qov') =
-    (==) qov qov'
-  (==) OpBinder{} _ = False
-
-  (==) (BinaryNoParensBinder b1 b2 b3) (BinaryNoParensBinder b1' b2' b3') =
-    (==) b1 b1' && (==) b2 b2' && (==) b3 b3'
-  (==) BinaryNoParensBinder{} _ = False
-
-  (==) (ParensInBinder b) (ParensInBinder b') =
-    (==) b b'
-  (==) ParensInBinder{} _ = False
-
-  (==) (NamedBinder _ ident b) (NamedBinder _ ident' b') =
-    (==) ident ident' && (==) b b'
-  (==) NamedBinder{} _ = False
-
-  (==) (PositionedBinder _ comments b) (PositionedBinder _ comments' b') =
-    (==) comments comments' && (==) b b'
-  (==) PositionedBinder{} _ = False
-
-  (==) (TypedBinder ty b) (TypedBinder ty' b') =
-    (==) ty ty' && (==) b b'
-  (==) TypedBinder{} _ = False
+  NullBinder == NullBinder =
+    True
+  (LiteralBinder _ lb) == (LiteralBinder _ lb') =
+    lb == lb'
```

Comment:
Not sure what the line-wrapping coding standard is. Could either:
* Use one line for each expression. Lines may grow long.
* Break after cutoff point, e.g. 80 characters. I feel this adds inconsistency to the structure.
* Break at each assignment. What's done in this PR.

Reply:
Breaking at each assignment like you've done here looks good to me. These things aren't really consistent across the compiler codebase at the moment.